src/strategies/information.rs

```diff
@@ -95,14 +95,20 @@ trait Question {
         self.acknowledge_answer(answer.value, hand_info, view);
     }
 }
-struct IsPlayable {
+
+struct CardHasProperty<T>
+    where T: Fn(&BoardState, &Card) -> bool
```

Comment:
maybe `type PropertyPredicate = Fn(&BoardState, &Card) -> bool` or something

Reply:
Ok! (Thanks for the free Rust lesson! :smile: )